sidekick/sidekick-phone-home/src/main/java/com/runsidekick/service/impl/PhoneHomeMetricServiceImpl.java

```diff
@@ -0,0 +1,86 @@
+package com.runsidekick.service.impl;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.runsidekick.model.EventType;
+import com.runsidekick.model.PhoneHomeMetric;
+import com.runsidekick.model.PhoneHomeMetricUtil;
+import com.runsidekick.service.PhoneHomeMetricService;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author yasin.kalafat
+ */
+@Service
+public class PhoneHomeMetricServiceImpl implements PhoneHomeMetricService {
+
+    private static final Logger LOGGER = LogManager.getLogger(PhoneHomeMetricServiceImpl.class);
+
+    @Value("${app.version:}")
+    private String appVersion;
```

Comment:
Is this app version or sidekick version? If it is Sidekick version, it is better to inject version into code or properties file during build and get it here automatically